Sources/MapboxDirections/RouteResponse.swift

```diff
@@ -26,6 +30,15 @@ public struct RouteResponse {
      This property does not persist after encoding and decoding.
      */
     public var created: Date = Date()
+    
+    /**
+     Managed of `RoadClasses` restrictions specified to `RouteOptions.roadClassesToAvoid` which were violated during route calculation.
+     
+     Routing engine may still utilize `RoadClasses` meant to be avoided in cases when routing is impossible otherwise.
+     
+     Resulting array is in the same order as `routes`, showing exact `RoadClasses` restrictions were ignored for each particular route at specific leg/step/intersection. `nil` and empty return arrays correspond to `nil` and empty `routes` array.
```

Comment:
This description is a bit confusing to me. Can arrays be in the same order if they are of different sizes?
If we want to describe the order 
```suggestion
     Violations are ordered by routes from the `routes` array, then by a leg, step, and intersection, where `RoadClasses` restrictions were ignored.
     `nil` and empty arrays correspond to `nil` and empty `routes` array.
```